source/Ship.cpp

```diff
@@ -492,6 +492,17 @@ void Ship::Load(const DataNode &node)
 			description += child.Token(1);
 			description += '\n';
 		}
+		else if( key == "remove" && child.Size() >= 2) {
```

Comment:
There's an extra space here before `key` and the parenthesis needs to be on a separate line.